pgctl/config.py

```diff
@@ -61,7 +57,7 @@ def from_file(self, filename):
                     result[key] = value
             return result
         elif filename.endswith(('.yaml', '.yml')):
-            return yaml_load(io.open(filename))
+            return yaml.safe_load(io.open(filename))
```

Comment:
this is (fwiw) the slow load -- you probably want to do the `Loader` trick to get the `CSafeLoader`

Reply:
```python
yaml.load(
    ...,
    Loader=getattr(yaml, 'CSafeLoader', yaml.SafeLoader),
)
```

Reply:
I wish this interface wasn't so bad :\

Reply:
same